internal/cli/alerts/unacknowledge.go

```diff
@@ -39,14 +39,17 @@ func (opts *UnacknowledgeOpts) initStore() error {
 	return err
 }
 
+var unackTemplate = `Alert '{{.ID}}' unacknowledged
+`
```

Comment:
[non-blocking] similar to the previous one
```suggestion
var unackTemplate = "Alert '{{.ID}}' unacknowledged\n"
```